lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -366,6 +366,14 @@ def _is_mpirun_for(cls, mpirun_path):
                     # next, check wheter version meets requirements (checked via _mpirun_version function)
                     version_var_name = 'EBVERSION' + mpiname.upper()
                     version = os.getenv(version_var_name)
+
+                    # mympirun is not compatible with OpenMPI version 2.0: this version contains a bug
+                    # see https://github.com/hpcugent/vsc-mympirun/issues/113
+                    if mpiname == "OpenMPI" and version_in_range(version, "2.0", "2.1"):
+                        LOGGER.error(("OpenMPI 2.0.x uses a different naming protocol for nodes. As a result, it isn't "
+                                      "compatible with mympirun. This issue is not present in OpenMPI 1.x and it has "
+                                      "been fixed in OpenMPI 2.1 and further."))
```

Comment:
missing exit?

test this with `foss/2017a`, `mympirun` should stop immediately after printing this error, there's no point in continuing